integration/framework/sql_helper.go

```diff
@@ -249,13 +249,16 @@ func (r *syncSQLRequest) Send() Awaitable {
 		defer rows.Close()
 
 		if !rows.Next() {
-			log.Warn("ReadBack:", zap.Error(err))
-			return
-		}
-		r.data, err = rowsToMap(rows)
-		if err != nil {
-			log.Warn("ReadBack", zap.Error(err))
-			return
+			// Upstream does not have the row
+			if r.requestType != sqlRequestTypeDelete {
+				log.Warn("ReadBack: no row, likely to be bug")
```

Comment:
What about `log.Fatal`?

Reply:
Making the process exit here makes cleaning up docker containers difficult. The test case will eventually timeout, and the framework will clean up all resources.